pkg/storage/ingest/reader_test.go

```diff
@@ -142,6 +142,52 @@ func TestPartitionReader_ConsumerError(t *testing.T) {
 	assert.Equal(t, [][]byte{[]byte("1"), []byte("2")}, records)
 }
 
+func TestPartitionReader_ConsumerStopping(t *testing.T) {
+	const (
```

Comment:
can you make this a `t.Parallel()`? It takes 1.5s and most of it is just sleeping